src/qfieldcloud_sdk/sdk.py

```diff
@@ -541,7 +550,9 @@ def download_file(
         download_type: FileTransferType,
         local_filename: Path,
         remote_filename: Path,
+        remote_md5sum: str,
```

Comment:
```suggestion
        remote_md5sum: str = None,
```
Why not optional? Also, missing from the docs below.